### PR TITLE
🚨🚨 Source LinkedIn Ads: update primary key for Analytics Streams

### PR DESCRIPTION
**Pull request by artem1205**
# What

Resolve https://github.com/airbytehq/oncall/issues/4863
Resolve https://github.com/airbytehq/oncall/issues/4595

### Another bug to be solved:

https://github.com/airbytehq/airbyte/pull/29175#issuecomment-1726814522

# How

update primary key; remove old properties from schema

### CODE REVIEW

**Review by tolik0 — APPROVED** [review]

File: airbyte-integrations/connectors/source-linkedin-ads/metadata.yaml [hunk 1]
```diff
@@ -29,6 +29,25 @@ data:
     oss:
       enabled: true
   releaseStage: generally_available
+  releases:
+    breakingChanges:
+      1.0.0:
+        message: This upgrade brings changes in primary key to *-analytics streams.
+        upgradeDeadline: "2024-04-25"
```
Comment (katmarkham) [inline airbyte-integrations/connectors/source-linkedin-ads/metadata.yaml:36]: lets push this to be April 30

File: docs/integrations/sources/linkedin-ads-migrations.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# LinkedIn Ads Migration Guide
+
+## Upgrading to 1.0.0
+
+Version 1.0.0 introduces changes in primary key for all *-analytics streams (including custom ones).
```
Comment (katmarkham) [inline docs/integrations/sources/linkedin-ads-migrations.md:5]: please add the stream names here as you did in the metadata file

**Review by katmarkham — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-linkedin-ads/metadata.yaml (hunk 1 above)
Comment (artem1205) [reply airbyte-integrations/connectors/source-linkedin-ads/metadata.yaml:36]: updated

File: docs/integrations/sources/linkedin-ads-migrations.md (hunk 2 above)
Comment (artem1205) [reply docs/integrations/sources/linkedin-ads-migrations.md:5]: updated

**Review by katmarkham — APPROVED** [review]

**Pull request merged**